README-Fabric.md

```diff
@@ -0,0 +1,9 @@
+# React Native Screens - Fabric version
+
+To use this library with your fabric application:
+1. Add latest react-native-screens
+3. iOS
```

Comment:
The ordered list numbers are off. Also, there should be info from which version fabric is supported,